Todo/TaskListsViewController.swift

```diff
@@ -218,12 +218,14 @@ class TaskListsViewController: UITableViewController, UISearchResultsUpdating {
     }
     
     func reloadIncompleteTasksCounts() {
-        var counts : [String : Int] = [:]
+        let counts : [String : Int] = [:]
         let rows = incompTasksCountsLiveQuery.rows
         while let row = rows?.nextRow() {
-            if let listId = row.key as? String, let count = row.value as? Int {
-                counts[listId] = count
-            }
+            print(row)
+            print(counts)
+            //if let listId = row.key as? String, let count = row.value as? Int {
+            //    counts[listId] = count
+            //}
```

Comment:
I think a temp workaround would be 'row.value(forKey: "key")'. Can you try that?